connector/github/github.go

```diff
@@ -16,12 +16,10 @@ import (
 	"strings"
 	"time"
 
+	"github.com/dexidp/dex/connector"
+	"github.com/sirupsen/logrus"
 	"golang.org/x/oauth2"
 	"golang.org/x/oauth2/github"
-
```

Comment:
❓ Would you mind much making this _three blocks_? Stdlib, third party, and anything starting with `github.com/dexidp/dex`? `goimports` has a setting for automatically doing this.

Reply:
sure!